--replay is broken, at least when run via docker & rig

Using the recommended rig approach to execute with --replay:

```
rig project create -- outrigger-drupal --replay
```

The system crashes as it starts attempting to generate the site.

```
     _-----_     ╭──────────────────────────╮
    |       |    │      Welcome to the      │
    |--(o)--|    │   fantabulous Outrigger  │
   `---------´   │     generator! 3.4.0     │
    ( _´U`_ )    ╰──────────────────────────╯
    /___A___\   /
     |  ~  |
   __'.___.'__
 ´   `  |° ´ Y `


Ok, I'm going to start assembling this project...
events.js:160
      throw er; // Unhandled 'error' event
      ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.join (path.js:1211:7)
    at child.packageJson (/usr/local/lib/node_modules/generator-outrigger-drupal/node_modules/generator-gadget/generators/app/index.js:220:51)
    at Object.<anonymous> (/usr/local/lib/node_modules/generator-outrigger-drupal/node_modules/yeoman-generator/lib/index.js:417:23)
    at /usr/local/lib/node_modules/generator-outrigger-drupal/node_modules/run-async/index.js:25:25
    at /usr/local/lib/node_modules/generator-outrigger-drupal/node_modules/run-async/index.js:24:19
    at /usr/local/lib/node_modules/generator-outrigger-drupal/node_modules/yeoman-generator/lib/index.js:418:9
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
[ERROR] Error running generator outrigger/generator outrigger-drupal --replay: 1
```

--- a/generators/app/index.js
+++ b/generators/app/index.js
@@ -130,6 +130,11 @@ module.exports = Generator.extend({
       delete store['resolved'];
       this.config.set(store);
 
+    }.bind(this));
+  },
+
+  configuring: {
+    sensibleDefaults: function() {
       options['skip-readme'] = true;
       // If using Docker-based environment defer running install locally.
       if (options['useENV']) {
@@ -145,7 +150,7 @@ module.exports = Generator.extend({
         // generator can set up theme wiring. This is the hard-coded name.
         options['themeName'] = 'patternlab';
       }
-    }.bind(this));
+    }
   },
 
   writing: {

